extending-valtimo/document/configuring-search-fields.md

```diff
@@ -49,20 +49,24 @@ To retrieve all the search fields created for a certain document definition use
 `GET /api/v1/document-search/{documentDefinitionName}/fields`. This will return an array of search fields.
 
 ## Update existing search field
-An existing search field can be updated with new values by calling the following endpoint:
+
+Existing search fields can be updated with new values by the endpoint below. The endpoint below accepts a list of search
+fields. The ordering in this list is adhered.
```

Comment:
adhered -> preserved? Not sure about this one, but I feel like the sentence doesn't entirely make sense right now.